starter-project-backend-g31/src/tests/setupdb.ts

```diff
@@ -21,7 +22,7 @@ export const connect = async () => {
 
 export const clear = async () => {
   const collections = await mongoose.connection.collections;
-
+  await UserProfile.deleteMany()
```

Comment:
remove userProfile from UserProfiletest.

Reply:
ok i used it because clear was not working